examples/conditional_format.pl

```diff
@@ -88,6 +96,51 @@
 
 ###############################################################################
 #
+# Example 9 (uses the same data as example 1)
```

Comment:
* Move Example 9 after Example 8. 
* Drop "uses the same data as example 1" in this comment.
* All comments should end with a full stop.